nil/core-concepts/shards-parallel-execution.mdx

```diff
@@ -9,7 +9,7 @@ Shards are smaller chains that are usually responsible for managing a portion of
 
 ## Execution shards
 
-All work related to processing transactions, storing accounts, and managing the local state occurs at the execution shard level. Execution shards are also tasked with passing message passing checks.
+All work related to processing transactions, managing cross-shard communications, storing accounts, and managing the local state occurs at the execution shard level. Execution shards are also tasked with passing message passing checks.
 
```

Comment:
It may be important to say that mempools are only used for external communication. Internal messages that are sent between shards don't go through the mempool. Their delivery mechanism is different. This distinction may be important for people coming from Ethereum, because they have a specific concept of mempool in mind (transactions in the mempool are not guaranteed to be processed in any way)

Reply:
I see, I've added this provision